src/database/config/config.js

```diff
@@ -0,0 +1,36 @@
+/*
+require('dotenv').config()
+
+module.exports = {
+  development: {
+    url: process.env.DEV_DATABASE_URL,
+    dialect: 'postgres',
+  },
+  test: {
+    url: process.env.TEST_DATABASE_URL,
+    dialect: 'postgres',
+  },
+  production: {
+    url: process.env.DATABASE_URL,
```

Comment:
Also here, remove this commented out section of code

Reply:
Yes I am going ahead and remove it